app/web/features/auth/useAuthStore.test.ts

```diff
@@ -1,8 +1,7 @@
-import { renderHook } from "@testing-library/react-hooks";
+import { act, renderHook } from "@testing-library/react-hooks";
 import { Empty } from "google-protobuf/google/protobuf/empty_pb";
 import { StatusCode } from "grpc-web";
 import { usePersistedState } from "platform/usePersistedState";
-import { act } from "react-test-renderer";
```

Comment:
This was the only place we were using `react-test-renderer` so I switched to use it from `testing-library` instead so we can remove that package.